<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowProjections</class>
 <widget class="QMainWindow" name="ShowProjections">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CADTool::Projections</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Isometric">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>360</y>
      <width>360</width>
      <height>320</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QSlider" name="zSlider">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>260</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="ySlider">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>210</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>220</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>    y-axis</string>
    </property>
   </widget>
   <widget class="QLabel" name="TopView">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>360</y>
      <width>360</width>
      <height>320</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QSlider" name="xSlider">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>160</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>680</y>
      <width>150</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Isometric Projection</string>
    </property>
   </widget>
   <widget class="QLabel" name="sideView">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>360</width>
      <height>320</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>680</y>
      <width>80</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Top View</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>270</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>    z-axis</string>
    </property>
   </widget>
   <widget class="QLabel" name="frontView">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>360</width>
      <height>320</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>330</y>
      <width>80</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Side View</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>330</y>
      <width>80</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Front View</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>170</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>    x-axis</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>310</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>25</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>320</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>     zoom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>690</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save File</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1266</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
